ngrx/ducks-schematics/src/collection/duck/schema.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "id": "SchematicsDuckSchematic",
+  "title": "ngrx-ducks Duck Options Schema",
+  "type": "object",
+  "description": "Creates a new Duck in the given or default project.",
+  "properties": {
+    "name": {
+      "type": "string",
+      "description": "The name of the Duck.",
+      "$default": {
+        "$source": "argv",
+        "index": 0
+      },
+      "x-prompt": "What name would you like to use for the Duck?"
+    },
+    "path": {
+      "type": "string",
+      "format": "path",
+      "description": "The path at which to create the Duck, relative to the workspace root.",
+      "visible": false
+    },
+    "project": {
+      "type": "string",
+      "description": "The name of the project.",
+      "$default": {
+        "$source": "projectName"
+      }
+    },
+    "noBarrel": {
+      "type": "boolean",
+      "default": false,
+      "description": "When false (the default) and not flat, creates a index.ts barrel file."
+    },
+    "flat": {
+      "type": "boolean",
+      "default": false,
+      "description": "When true, creates files at the top level of the project."
+    },
+    "skipTests": {
```

Comment:
I would rename it to `spec`.
The default value should be true.

This follows the pattern of the existing schematics.

Reply:
Somehow I mixed that up 😄